src/test/java/io/github/sskorol/data/DataSuppliers.java

```diff
@@ -28,8 +32,26 @@
     public <T> T[] getDataCollection(final Method method) {
         return (T[]) ofNullable(method.getAnnotationsByType(Data.class))
                 .map(data -> StreamEx.of(data)
-                        .flatMap(dataIn -> Arrays.stream(readerOf(dataIn.source())
+                        .flatMap(dataIn -> stream(readerOf(dataIn.source())
                                 .readFrom(dataIn.source(), (Class<T>) dataIn.entity()))).toArray())
+                .orElseThrow(() -> new NoClassDefFoundError("Data class not found"));
```

Comment:
`NoClassDefFoundError` is more about physical class' missing.
You may want to throw smth like `IllegalStateException`.